### DISCUSSION TITLE
Meaning to  ←[2K in output lines?

### DISCUSSION THREAD

**Discussion by Patch4747** [ggml-org/whisper.cpp#1569] [General]
After using the `stream` command, each output line of text is appended with `←[2K`. I'm wondering what it means, if anything? https://github.com/ggerganov/whisper.cpp/blob/6559b538e5e05cfa199c15d46ca5bd0edd353974/examples/stream/stream.cpp#L345

**Comment by tamo** [comment]
https://stackoverflow.com/questions/1508490/erase-the-current-printed-console-line

maybe you can use a vt100 compatible terminal
